### PR TITLE
Add ShiftStack ingress-perf Periodic CI Job

### PR DESCRIPTION
**Pull request by MiguelCarpio**
Adding ingress performance test Periodic CI Job in shiftstack-ci.

### CODE REVIEW

**Comment by MiguelCarpio** [comment]
Please wait to merge it; I want to perform a test first.

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml [hunk 1]
```diff
@@ -50,6 +50,13 @@ tests:
     - chain: ipi-openstack-pre
     - ref: fips-check
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ingress-perf
+  minimum_interval: 72h
+  steps:
+    cluster_profile: openstack-vexxhost
+    env:
+      INFRA_NODE_TYPE: ci.m1.xlarge
```
Comment (EmilienM) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml:58]: why did you add `INFRA_NODE_TYPE` ? It's only useful in the `ipi-conf-inframachineset` step registry which isn't used in our workflows (a grep shows it's used by `openshift-upgrade-aws-infra-workflow` only).

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml [hunk 2]
```diff
@@ -50,6 +50,13 @@ tests:
     - chain: ipi-openstack-pre
     - ref: fips-check
     workflow: openshift-e2e-openstack-ipi
+- as: e2e-openstack-ingress-perf
+  minimum_interval: 72h
```
Comment (EmilienM) [inline ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml:54]: Probably too frequent for a perf test.

File: ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml [hunk 3]
```diff
@@ -7563,6 +7563,87 @@ periodics:
...
+    repo: shiftstack-ci
+  labels:
+    ci-operator.openshift.io/cloud: openstack-vexxhost
+    ci-operator.openshift.io/cloud-cluster-profile: openstack-vexxhost
+    ci-operator.openshift.io/variant: periodic-4.16
+    ci.openshift.io/generator: prowgen
+    job-release: "4.16"
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  minimum_interval: 72h
+  name: periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf
```
Comment (EmilienM) [inline ci-operator/jobs/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main-periodics.yaml:7583]: We need notifications on Slack like other periodic jobs.

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml (hunk 1 above)
Comment (MiguelCarpio) [reply ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml:58]: Because the [openshift-qe-ingress-perf](https://github.com/openshift/release/blob/master/ci-operator/step-registry/openshift-qe/ingress-perf/openshift-qe-ingress-perf-ref.yaml) step, which uses the [Ingress-perf wrapper](https://github.com/cloud-bulldozer/e2e-benchmarking/tree/master/workloads/ingress-perf), requires at least two workers with the "infra" role/label to perform the OCP Ingress performance test.

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml (hunk 2 above)
Comment (MiguelCarpio) [reply ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml:54]: Yes, I realized it; I'm working to set up an appropriate interval.

File: ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml (hunk 1 above)
Comment (EmilienM) [reply ci-operator/config/shiftstack/shiftstack-ci/shiftstack-shiftstack-ci-main__periodic-4.16.yaml:58]: ok, makes sense. The problem is available resources in VEXXHOST; which is why we need to not run that too often.

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by EmilienM** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by EmilienM** [comment]
not sure why the job hasn't kicked off

**Comment by EmilienM** [comment]
running now!

https://prow.ci.openshift.org/view/gs/test-platform-results/pr-logs/pull/openshift_release/51202/rehearse-51202-periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf/1783584187526156288

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
The last job failed at the ingress-perf test with the message: "Error creating prometheus client". Reason: It got a "server_error: server error: 503" on the ElasticSearch side. Let's test it again.

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
/pj-rehearse periodic-ci-shiftstack-shiftstack-ci-main-periodic-4.16-e2e-openstack-ingress-perf

**Comment by MiguelCarpio** [comment]
Closed by error

**Comment by MiguelCarpio** [comment]
Closed by error
